examples/catalog/lib/animated_list.dart

```diff
@@ -196,9 +196,7 @@ class CardItem extends StatelessWidget {
   }
 }
 
-void main() {
-  runApp(AnimatedListSample());
-}
+void main() { runApp(AnimatedListSample()); }
```

Comment:
this is a surprising change

Reply:
I'd probably have an exception to this rule for top-level members.

Reply:
Exception added for top level functions.